provider/azure/azure_discover.go

```diff
@@ -28,7 +29,16 @@ func (p *Provider) Help() string {
    tenant_id:         The id of the tenant
    client_id:         The id of the client
    subscription_id:   The id of the subscription
-   secret_access_key: The authentication credential
+	 secret_access_key: The authentication credential
```

Comment:
Somehow the indentation is different than the other things. Maybe tabs vs spaces?

Reply:
Fixed! 👍